main.rb

```diff
@@ -22,14 +22,16 @@ def runCommand(command)
       raise stderr_str
     end
 end
-runCommand("brew tap tuist/tuist && brew install tuist")
-options = ""
-tuist_path = env_has_key("AC_TUIST_PATH")
-project_only = env_has_key("AC_TUIST_PROJECT_ONLY") || "false"
-if tuist_path
-  options += " -p #{tuist_path}"
-end
-if project_only == "true"
-  options += " -P"
+
+runCommand("brew tap tuist/tuist")
+tuist_path = env_has_key("AC_TUIST_PATH") || abort("Missing AC_TUIST_PATH")
+tuist_version = env_has_key("AC_TUIST_VERSION")
+spec_version = ""
+
+if tuist_version
+  spec_version = "@#{tuist_version}"
 end
-runCommand("tuist generate #{options}")
\ No newline at end of file
+  
+runCommand("brew install tuist#{spec_version}")
```

Comment:
You can leave it like this, but you can also update it as follows:
```suggestion
runCommand("brew install tuist#{spec_version} && tuist generate -p #{tuist_path}")
```

Reply:
I did not use it on the same line because it gives different outputs and the outputs appear separately.

I can change it, this time the output of both install and generate commands will appear as one, making it difficult to read the log. Am I wrong?